xbmc/cores/VideoSettings.h

```diff
@@ -101,7 +101,6 @@ class CVideoSettings
   int m_SubtitleStream;
   float m_SubtitleDelay;
   bool m_SubtitleOn;
-  bool m_SubtitleCached; // not used -> remove from DB
```

Comment:
This line doesn't seem relevant to this PR/commit. Can you move it to a new commit?

Reply:
Moved to separate commit: https://github.com/xbmc/xbmc/pull/18749/commits/ed0f2aa83a3e6967630b5ea996b2c48a010856d2

Reply:
Thanks!